app/lib/theme/subthemes/icon_button_theme.dart

```diff
@@ -7,31 +7,29 @@ class FlexIconButtonTheme {
 
   static IconButtonThemeData lightIconButtonTheme = IconButtonThemeData(
     style: IconButton.styleFrom(
-      elevation: 1,
-      shadowColor: FlexAppColorScheme.lightScheme.primary,
-      foregroundColor: FlexAppColorScheme.lightScheme.primary,
+      elevation: 0,
       backgroundColor: FlexAppColorScheme.lightScheme.onPrimary,
+      foregroundColor: FlexAppColorScheme.lightScheme.primary,
       disabledForegroundColor: FlexAppColorScheme.lightScheme.disabled,
-      disabledBackgroundColor: FlexAppColorScheme.lightScheme.background,
-      padding: const EdgeInsets.all(FlexSizes.xs),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(FlexSizes.borderRadiusSm),
+      disabledBackgroundColor: FlexAppColorScheme.lightScheme.onBackground,
```

Comment:
on as in on and off?

I'm used to "on" being reserved for events 🤷 would suggest something like "activeBackground"